.github/workflows/auto-start-ci.yml

```
@@ -13,6 +13,7 @@ on:
 
 jobs:
   startCI:
+    if: github.repository == 'nodejs/node'
```

```suggestion
    if: secrets.JENKINS_USER && secrets.JENKINS_TOKEN
```

Also not sure if it works. Feel free to try on nodejs/node-auto-test first (I can re-enable Actions there, just give me a ping).

---

It might need to be

```suggestion
    if: ${{ secrets.JENKINS_USER && secrets.JENKINS_TOKEN }}
```

https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#about-contexts-and-expressions

---

https://github.community/t/jobs-job-id-if-does-not-work-with-env-secrets/16928/5 It doesn't sound like it does work...